### DISCUSSION TITLE
请问一下各位老铁，keda可不可以符合多个条件之后再进行扩展？

### DISCUSSION THREAD

**Discussion by JoeyX-u** [kedacore/keda#1476] [Q&A / Need Help]
请问一下各位老铁，keda可不可以符合多个条件之后再进行扩展，比如说必须同时满足内存和cpu的阈值后再进行扩展

**Comment by ringtail** [comment]
> 请问一下各位老铁，keda可不可以符合多个条件之后再进行扩展，比如说必须同时满足内存和cpu的阈值后再进行扩展

你说的这个需求，使用标准HPA即可。

**Reply by JoeyX-u** [reply]
> > 请问一下各位老铁，keda可不可以符合多个条件之后再进行扩展，比如说必须同时满足内存和cpu的阈值后再进行扩展
> 
> 你说的这个需求，使用标准HPA即可。

主要是想利用Prometheus的指标来做，因为扩展不仅仅是单一指标来判断，所以想问下keda看了半天没发现可以实现这种需求

**Reply by ringtail** [reply]
> 主要是想利用Prometheus的指标来做，因为扩展不仅仅是单一指标来判断，所以想问下keda看了半天没发现可以实现这种需求

一样的，在HPA这个层面上，keda是包裹了传统的HPA，没有额外的增强
